Backend/BikesRentalServer.Tests/ServicesTests/UsersServiceTests/GetAllTechs.cs

```diff
@@ -0,0 +1,101 @@
+﻿using BikesRentalServer.Models;
+using BikesRentalServer.Services;
+using FluentAssertions;
+using System.Collections.Generic;
+using Xunit;
+
+namespace BikesRentalServer.Tests.ServicesTests.UsersServiceTests
+{
+    public class GetAllTechs : UsersServiceTestsBase
+    {
+        [Fact]
+        public void GetAllTechsShouldReturnEmptyListWhenNoTechs()
+        {
+            UsersRepository.Setup(r => r.GetAllTechs()).Returns(new List<User>());
+
+            var usersService = GetUsersService();
+            var result = usersService.GetAllTechs();
+
+            result.Status.Should().Be(Status.Success);
+            result.Object.Should().BeEmpty();
+        }
+
+        [Fact]
+        public void GetAllTechsReturnsAllExistingTechs()
+        {
+            var techs = new[]
+            {
+                new User
+                {
+                    Role = UserRole.Tech,
+                    Status = UserStatus.Active,
+                    Username = "wojtek",
+                },
+                new User
+                {
+                    Role = UserRole.Tech,
+                    Status = UserStatus.Active,
+                    Username = "kasia",
+                },
+                new User
+                {
+                    Role = UserRole.Tech,
+                    Status = UserStatus.Active,
+                    Username = "kacper",
+                },
+            };
+            UsersRepository.Setup(r => r.GetAllTechs()).Returns(techs);
+
+            var usersService = GetUsersService();
+            var result = usersService.GetAllTechs();
+
+            result.Status.Should().Be(Status.Success);
+            result.Object.Should().BeEquivalentTo(techs);
+        }
+
+        [Fact]
+        public void GetAllTechsDontIncludeAdminAndNormalUsers()
```

Comment:
Ten test nie jest potrzebny, nie działa zresztą ;)

Reply:
Możnaby go połączyć z poprzednim, ale jednak warto testować czy zwracasz tylko techów

Reply:
To repozytorium za to odpowiada

Reply:
Hmmm. A `GetAllUsers` nie ma takiego testu?

Reply:
wygląda jakby .where w repo wgl nic nie robił

Reply:
Where w repo pewnie działa, ale w tym teście nie wywołujesz metody repo tylko mocka. A ten mock ma powiedziane by zwrócić dziką chmarę userów zamiast poprawnego wyniku (i pewnie taki sam jest dla userów)